Loop/Managers/LoopAlertsManager.swift

```diff
@@ -48,7 +48,7 @@ extension LoopAlertsManager: CBCentralManagerDelegate {
     private func onBluetoothPermissionDenied() {
         log.default("Bluetooth permission denied")
         let content = Alert.Content(title: NSLocalizedString("Bluetooth Permission Denied", comment: "Bluetooth permission denied alert title"),
-                                      body: NSLocalizedString("You must accept permission to use bluetooth to receive alerts, alarms or sensor glucose readings, and communicate with your pump.",
+                                      body: NSLocalizedString("The Tidepool Loop app needs permission to access your iPhone’s Bluetooth connection in order for the app to communicate with your pump and CGM sensor. You will be unable to use the app to receive CGM information and send commands to your pump until Bluetooth permissions are enabled.",
```

Comment:
This cannot be accepted back into DIY Loop.  It would be better if the app name was left out, used generically ("Loop"), or substituted using plist data.